### ISSUE TITLE
[5.0 beta] Dupicate themes

### ISSUE THREAD

**Issue by printingotb** [Ultimaker/Cura#11873]
### Application Version

5.0 beta

### Platform

win10

### Printer

custom fff

### Reproduction steps

Open dropdown to select Themes

### Actual results

There are two of each theme in the list.

### Expected results

Should be only one

### Checklist of files to include

- [ ] Log file
- [ ] Project file

### Additional information & file uploads

![duplicate Themes](https://user-images.githubusercontent.com/32172100/164665647-992e5797-1766-41eb-947e-6aaec5facddf.jpg)

**Comment by Vandresc** [comment]
Yes, we are currently working on this issue, should be fixed for the stable. Thank you for your feedback and for giving the beta a try.

**Comment by printingotb** [comment]
Happy to help, keep up the good work!

**Comment by fvrmr** [comment]
Devs see CURA-9147

**Comment by nallath** [comment]
Fixed for stable (we just forgot to close this, sorry!)
